### PR TITLE
Fix highlight on nav and open 'internal links' in same window

### PR DESCRIPTION
**Pull request by ericfranz**

### CODE REVIEW

File: apps/dashboard/app/apps/ood_app.rb [hunk 1]
```diff
@@ -42,25 +42,42 @@ def title
...
+      app_type: type,
+      app_owner: owner,
+      app_name: name,
+      app_token: token
+    }
+  end
+
+  def internal_url?
+    custom_url && self.class.relative_url?(custom_url)
+  end
```
Comment (johrstrom) [inline apps/dashboard/app/apps/ood_app.rb:69]: I feel like some test cases are in order. Shouldn't an app with no `custrom_url` (manifest.url) be internal here?

Comment (ericfranz) [reply apps/dashboard/app/apps/ood_app.rb:69]: An app with no custom url is always external

Comment (ericfranz) [reply apps/dashboard/app/apps/ood_app.rb]: I'll push that later with tests. Just keep this PR for the cosmetic fix.

**Review by johrstrom — APPROVED** [review]
👍

**Pull request merged**